src/ReactiveUI/ReactiveList.cs

```diff
@@ -568,7 +563,13 @@ public bool ChangeTrackingEnabled
 
         public IDisposable SuppressChangeNotifications()
         {
-            Interlocked.Increment(ref _resetNotificationCount);
+            NotifyCollectionChangedEventArgs resetEventArgs = null;
+
+            if (Interlocked.Increment(ref _resetNotificationCount) == 1) {
+                // Changes were not suppressed before, publish a Changing reset notification.
+                resetEventArgs = new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Reset);
```

Comment:
Is there a reason to not use the reset/publishResetNotification in here? Wondering if we can reduce the breaking changes to a minimum.

Reply:
Note that with this patch only a `Changed`, but no `Changing` notification, is pushed in the disposable. Using `publishResetNotification()` in here would cause a duplicate `Changing` notification after the changes have been made, which would be unexpected but perhaps less breaking. I removed `publishResetNotification()` because it was used only in `Reset()` and `SuppressChangeNotifications()` and since it is no longer used there I feel like it would be better to have a compiler error rather than unexpected behaviour when overriding this method.

I guess it comes down to this: Do you prefer a duplicate, technically incorrect, `Changing` notification but no breaking changes? Or would you rather have the (small) breaking change and no duplicate notifications?